include/zephyr/net/socket.h

```diff
@@ -164,13 +164,15 @@ struct zsock_pollfd {
  *  This option accepts any value.
  */
 #define TLS_SESSION_CACHE_PURGE 13
-
 /** Socket option to set DTLS Connection ID to be used for the DTLS session.
  *  The option accepts an byte array, holding the CID to use.
  *  Setting an empty CID (option length set to 0) indicates that the socket is
  *  willing to handle CID from a peer, but does not specify its own CID.
  */
 #define TLS_DTLS_CONNECTION_ID 14
+/** Socket option to get DTLS Connection ID from peer.
+ */
```

Comment:
The documentation could be a bit more verbose:
- specify that the option is read-only,
- specify the expected data format.

Reply:
Doesn't seem to be resolved?

Reply:
I had not yet pushed.  It's updated now.